www/artists/afyfe/index.html

```diff
@@ -24,7 +24,7 @@
             </div>
             <div id="footer">
                 <p>
-                    Copyright 2008 Amber Fyfe. All images are the property of Amber Fyfe.
+                    "It's not the same feel," said Fyfe. "You only have what the other artist did to look at; you don't have his inspiration or what inspired him."                    <br>
```

Comment:
Same sort of thing here - it feels out of place on a website. Don't just blindly copy/paste - reformat it to fit the website, and add proper attribution at the end